tools/bin/commands/integration-test

```diff
@@ -24,6 +24,10 @@ integration-test::run() {
     call_maven "$(maven_args)" ":test-parent,:test-support,:integration-test"
 }
 
+integration-test::init() {
+    echo "Running integration tests"
+}
+
```

Comment:
Is this required?

Reply:
We needed some init function. If we remove it, the syndesis command fails.